tool/install_flex_dependencies.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+cd  flex
+npm install
+npm install @inexor-game/connector
```

Comment:
This'll only be problematic if you change the modules after install time which is usually not the case for end-users.